python-avd/pyavd/_eos_designs/schema/schema_fragments/generate_cv_tags.schema.yml

```diff
@@ -18,6 +18,10 @@ keys:
         type: bool
         default: false
         description: Enable the generation of CloudVision Topology Tags (hints).
+      campus_fabric:
+        type: bool
+        default: false
+        description: Generate CloudVision device and interface Topology Tags for Campus fabric devices.
```

Comment:
This should be PREVIEW

Reply:
Add more info about the feature here including the various limitations, like only including devices where certain keys have been set and only for topology type blah blah.